### ISSUE TITLE
Particle-particle heat transfer in thermal DEM

### ISSUE THREAD

**Issue by wuhao1938** [PhasicFlow/phasicFlow#98]
Will the heat transfer, such as conduction and thermal radiation,  be enabled in the the future version?
Particle-particle conduction in TDEM is rather easy by adding a thermal contact resistance given in the well-known reference.

References may be useful:
1. https://doi.org/10.1016/j.icheatmasstransfer.2020.104829
2. https://doi.org/10.1016/j.ijheatmasstransfer.2021.121432

**Comment by PhasicFlow** [comment]
Yes, we are planning to add heat transfer to both DEM and CFD-DEM parts. This heat transfer will include particle-particle, particle-fluid and radiation heat transfer. 
Thanks for suggestion on articles. They are very good.
